src/rep.rs.in

```diff
@@ -350,13 +350,6 @@ impl GistOptions {
 }
 
 #[derive(Debug, Deserialize)]
-pub struct Permissions {
```

Comment:
I had this over here because I think Repo's have a `permissions` object is sometimes [serialized](https://developer.github.com/v3/repos/#get) but I'm not sure if they always are. When I can confirm I'd like that to live in this module.